### PR TITLE
[ADMIN] add new entries for 5 obj reservation placeholders (10498-10502) for SEW

### PR DESCRIPTION
**Pull request by gocadimic**
> Note:

**In making a submission to the Open Mobile Alliance Registry, you understand and agree that your submission is made under the BSD-3 Clause License as set forth on the Open Mobile Alliance Registry and that additional formats of your submission may be created by the Open Mobile Alliance tools and processes, which may convert the content of your submission, and that the Open Mobile Alliance bears no liability for the additional formats of your submission nor for any conversion of the content of your submission.**

### CODE REVIEW

**Comment by gocadimic** [comment]
@workgreg,
The DDF.xml file was updated with new placeholders obj-10498-10502 and customized as requested.
All changes have been made to the new branch `sew-obj-reserve-placeholders`. Submitting the new objects should be continued there.
A fully qualified 10498.xml, 10499.xml, 10500.xml, 10501.xml, and 10502.xml files should be submitted to the given branch for version 1.0. Please make sure that the content of the DDF.xml matches the request in [PR #783](https://github.com/OpenMobileAlliance/lwm2m-registry/pull/783).
Once the new Objects are submitted, the same 10498.xml, 10499.xml, 10500.xml, 10501.xml, and 10502.xml files have to be updated to the version_history folder, with the same content as in DDF.xml, named  10498-1_0.xml....10502-1_0.xml.
The final step would be creating a PR against the prod branch.
For more explanation, see [README.md](https://github.com/OpenMobileAlliance/lwm2m-registry#readme).

File: DDF.xml [hunk 1]
```diff
@@ -6358,6 +6358,76 @@ The Object also allows steering the device towards a particular PLMN, or, to a p
...
     </Item>
+    <Item>
+        <ObjectID>10498</ObjectID>
+        <URN>urn:oma:lwm2m:x:10498</URN>
+        <Name>TBD</Name>
+        <Description>Reservation 10498</Description>
+        <Owner>TBD</Owner>
+        <Source>2</Source>
+        <Ver>1.0</Ver>
+        <DDF>10498.xml</DDF>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <DDF></DDF>
```

File: DDF.xml [hunk 2]
```diff
@@ -6358,6 +6358,76 @@ The Object also allows steering the device towards a particular PLMN, or, to a p
...
+    </Item>
+    <Item>
+        <ObjectID>10499</ObjectID>
+        <URN>urn:oma:lwm2m:x:10499</URN>
+        <Name>TBD</Name>
+        <Description>Reservation 10499</Description>
+        <Owner>TBD</Owner>
+        <Source>2</Source>
+        <Ver>1.0</Ver>
+        <DDF>10499.xml</DDF>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <DDF></DDF>
```

File: DDF.xml [hunk 3]
```diff
@@ -6358,6 +6358,76 @@ The Object also allows steering the device towards a particular PLMN, or, to a p
...
+    </Item>
+    <Item>
+        <ObjectID>10500</ObjectID>
+        <URN>urn:oma:lwm2m:x:10500</URN>
+        <Name>TBD</Name>
+        <Description>Reservation 10500</Description>
+        <Owner>TBD</Owner>
+        <Source>2</Source>
+        <Ver>1.0</Ver>
+        <DDF>10500.xml</DDF>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <DDF></DDF>
```

File: DDF.xml [hunk 4]
```diff
@@ -6358,6 +6358,76 @@ The Object also allows steering the device towards a particular PLMN, or, to a p
...
+    </Item>
+    <Item>
+        <ObjectID>10501</ObjectID>
+        <URN>urn:oma:lwm2m:x:10501</URN>
+        <Name>TBD</Name>
+        <Description>Reservation 10501</Description>
+        <Owner>TBD</Owner>
+        <Source>2</Source>
+        <Ver>1.0</Ver>
+        <DDF>10501.xml</DDF>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <DDF></DDF>
```

File: DDF.xml [hunk 5]
```diff
@@ -6358,6 +6358,76 @@ The Object also allows steering the device towards a particular PLMN, or, to a p
...
+    </Item>
+    <Item>
+        <ObjectID>10502</ObjectID>
+        <URN>urn:oma:lwm2m:x:10502</URN>
+        <Name>TBD</Name>
+        <Description>Reservation 10502</Description>
+        <Owner>TBD</Owner>
+        <Source>2</Source>
+        <Ver>1.0</Ver>
+        <DDF>10502.xml</DDF>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <DDF></DDF>
```

**Review by jpradocueva — COMMENTED** [review]
In Production we cannot have a reference to Objects that are not available. The DDF, must be empty and DDFLink set to "0"

**Pull request merged**